src/Framework/FieldsAPI/FormField/FieldStoreAsMeta.php

```diff
@@ -17,8 +17,8 @@ trait FieldStoreAsMeta {
 	 * @unreleased
 	 * @return $this
 	 */
-	public function storeAsDonorMeta() {
-		$this->storeAsDonorMeta = true;
+	public function storeAsDonorMeta( $storeAsDonorMeta = true ) {
+		$this->storeAsDonorMeta = $storeAsDonorMeta;
```

Comment:
```suggestion
		$this->storeAsDonorMeta = (bool) $storeAsDonorMeta;
```

Reply:
Yeah, that makes sense. I'll get it updated.

Reply:
@JasonTheAdams Updated with a test, see https://github.com/impress-org/givewp/pull/5670/commits/9a3753e819c555707f4fe7373862d001d5c28cc5.